third-party-code.rst

```diff
@@ -0,0 +1,253 @@
+
+*************
+ Definitions
+*************
+
+**PHP tooling**
+   Refers to the code behind the PHP.net website, the documentation generator
+   project PhD, the PHP wiki, and other similar systems. In general, “PHP code
+   run by PHP.net.”
+
+**Documentation**
+   Refers to objective information about PHP, the PHP language, the PHP standard
+   library, and PHP ecosystem hosted on PHP.net. This may include reference
+   material, tutorials, FAQs, and similar.
+
+**Marketing material**
+   Refers to content on PHP.net or similar sites intended to promote or
+   evangelize PHP the language or ecosystem.
+
+**Libraries**
+   Refers to existing third party code packages or tools, either C extensions or
+   PHP code, maintained by someone other than the PHP Internals team. It also
+   includes command line utilities used primarily by a developer. It may also
+   refer to non-profit PHP ecosystem organizations, such as the PHP Foundation
+   or PHP-FIG.
+
+**Web Application**
+   Refers to a “full” web framework that provides end-to-end web application
+   capabilities, or an installable complete application. It does not refer to
+   command line utilities used primarily by developers building applications.
+
+**Approved license**
+   Refers to a license `approved by the Free Software Foundation as Free
+   Software <https://www.gnu.org/licenses/license-list.en.html>`_, and that is
+   inbound compatible with GPLv3.
+
+*********************
+ Approval Heuristics
+*********************
+
+PHP Tooling
+===========
+
+PHP tooling MAY make use of third party libraries, provided that the library
+meets all of the “Inclusion” criteria, and does not meet any of the “Exclusion”
+criteria.
+
+Inclusion criteria
+------------------
+
+#. The library must have a stable >= 1.0 release, and have had one for at least
+   a year. (This is to ensure it has longevity.)
+#. The library provides targeted, necessary functionality.
+#. The library is a recognized de facto standard, or one of a small number of de
+   facto standards, in its problem space.
+#. The library is available under an Approved License.
+
+Exclusion criteria
+------------------
+
+#. The library is a Web Application
+#. The library is not available under an Approved License.
+#. The library has shown no meaningful activity for one year prior to its first
+   inclusion.
+
+PHP tooling maintainers MAY use their judgement to determine if a library meets
+the above criteria, but SHOULD be conservative in their interpretation of
+whether or not a library satisfies the necessary criteria.
+
+Explicitly approved libraries
+-----------------------------
+
+The following libraries have been explicitly approved by RFC vote.
+
+-  Composer
+-  Xdebug
+-  phpunit/phpunit
+-  phpstan/phpstan
+-  vimeo/psalm
+-  michelf/php-markdown
+-  phpmailer/phpmailer
+-  squizlabs/php_codesniffer
+-  friendsofphp/php-cs-fixer
+-  symfony/dotenv
+-  symfony/console
+-  fzaninotto/faker
```

Comment:
This one is archived. As far as I know the project is continued by https://github.com/FakerPHP/Faker

Reply:
It's only used by web-pecl, which is on its way to being archived, anyway. I think it would be fine to not list it and someone can get whatever the current project is down the road if they want to use it then.